### ISSUE TITLE
Morph glitching out after MC name change

### ISSUE THREAD

**Issue by Onemario1234** [iChun/Morph#834]
So, I had recently changed my name from onemario12345 to Onemario4321, and I also changed my skin aswell, and after doing so I noticed Morph bugged out, and seperated my newly named and skinned self from my previously named and skinned self. I was able to morph between them, and I couldn't delete my previous self, as it still thinks that was my main unmorphed form. I was fine with this at first, but I tried to enter a Sync Chamber, and it acted like I didn't have any other shells, when I had 2 others. I also noticed one of my shells had my previous skin. When I removed the Shell Storage that had my previous skin in it, the game crashed. I can't delete this world because I built so much in it. I'm using Minecraft version 1.7.10, the most recent version of Morph, but not the most recent versions of iChunUtil, or Sync. I'm going to do so soon.

EDIT: As of writing this, I have installed the newest versions of Sync and iChunUtil, but I haven't tried it again, yet.

**Comment by iChun** [comment]
This was an issue with the way old Morph stores names/player morphs. It'll
be sorted in the later versions.

On 24 Feb 2017 1:06 pm, "Onemario1234" <notifications@github.com> wrote:

So, I had recently changed my name from onemario12345 to Onemario4321, and
I also changed my skin aswell, and after doing so I noticed Morph bugged
out, and seperated my newly named and skinned self from my previously named
and skinned self. I was able to morph between them, and I couldn't delete
my previous self, as it still thinks that was my main unmorphed form. I was
fine with this at first, but I tried to enter a Sync Chamber, and it acted
like I didn't have any other shells, when I had 2 others. I also noticed
one of my shells had my previous skin. When I removed the Shell Storage
that had my previous skin in it, the game crashed. I can't delete this
world because I built so much in it. I'm using Minecraft version 1.7.10,
the most recent version of Morph, but not the most recent versions of
IChunUtil, or Sync. I'm going to do so soon.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/iChun/Morph/issues/834>, or mute the thread
